composer.json

```diff
@@ -6,11 +6,28 @@
     {
         "type" : "pear",
         "url" : "https://pear.php.net"
+    },
+    {
+        "type": "package",
+        "package": {
+            "name": "PHPOffice/PHPExcel",
+            "version": "1.8.0",
+            "source": {
+                "url": "https://github.com/PHPOffice/PHPExcel.git",
+                "type": "git",
+                "reference": "1.8.0"
+            },
+            "autoload": {
+                "psr-0": {
+                    "PHPExcel": "Classes/"
+                }
+            }
+        }
```

Comment:
@gluneau You might want to add the note to the documentation about now php-xmlwriter is required to successfully go through composer installation process on minimal CenOS 6 (which I did not notice before this latest update 16.02). ![Alt text](https://monosnap.com/file/jGv9bsA5jeH4AKO4gqjTaJmB8Kr8KV.png).